zuko/transforms.py

```diff
@@ -743,6 +731,34 @@ def _inverse(self, y: Tensor):
         return x
 
 
+class BoundedBernsteinTransform(BernsteinTransform):
```

Comment:
Add a docstring. No need for a long description (unless you want to explain the math), but say that the $[-B, B]$ domain interval is mapped to the $[-B, B]$ co-domain interval.